third_party/move/move-compiler-v2/tests/checking/typing/lambda.exp

```diff
@@ -24,7 +24,7 @@ error: cannot use `()` with an operator which expects a value of type `u64`
 56 │             i = i + action(XVector::borrow(v, i)); // expected to have wrong result type
    │                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-error: cannot return `u64` from a function with result type `|integer| with copy+store`
+error: expected function value `|integer| has ..` but found `u64`
```

Comment:
nit: "function value `|integer| has ..`" is a bit confusing. "function value of type `|integer|` might be better, if it is not too cumbersome.